requirements-dev.txt

```diff
@@ -7,3 +7,4 @@ pytest-cov>=4.0.0
 pytest-asyncio>=0.15.1
 aiounittest>=1.4.2
 grpc-interceptor>=0.15.4
+ruff
```

Comment:
I'd expect formatters / linters to be generally backwards compatible - but - doesn't hurt to be careful, and it would be consistent with the rest of the reqs.